arm-containerregistry/2017-03-01/examples/RegistryDelete.json

```diff
@@ -0,0 +1,12 @@
+{
+    "parameters": {
+        "api-version": "2017-03-01",
+        "subscriptionId": "subid",
+        "resourceGroupName": "rgname",
+        "registryName": "registry"
+    },
+    "responses": {
+        "200": {},
```

Comment:
In case you are cleaning up the storage account once registry is gone, this operation could be long running . Better to make this asynchronous as well.